packages/did-ethr-resolver/test/did-operator.volta.test.ts

```diff
@@ -0,0 +1,50 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Keys } from '@ew-did-registry/keys';
+import {
+  ProviderTypes,
+  ProviderSettings,
+} from '@ew-did-registry/did-resolver-interface';
+import { Methods } from '@ew-did-registry/did';
+import { utils, Wallet, providers } from 'ethers';
+import { Operator, ethrReg, VoltaAddress1056 } from '../src';
+import { EwPrivateKeySigner, IdentityOwner } from '../src/implementations';
+
```

Comment:
@jrhender what if we ensure tha on success, the document creation has the correct return ?

```suggestion
import { expect } from 'chai';

```